flink-ai-flow/ai_flow/api/ops.py

```diff
@@ -176,22 +191,27 @@ def predict(input: Union[Channel, List[Channel]],
             output_num=1,
             name: Text = None) -> Union[Channel, Tuple[Channel]]:
     """
-    Predictor Operator. Do prediction job with the specific model version.
-
-    :param input: List of Channel. It contains the dataset data used in prediction.
-    :param model_info: Information about the model which is in prediction. Its type can be ModelMeta
-                       of py:class:`ai_flow.meta.model_meta.ModelMeta` or Text or int. The model_info
-                       means name in the metadata service when its type is Text and it means id when its type is
-                       int. The ai flow will get the model meta from metadata service by name or id.
-    :param prediction_processor: The user defined function in predict operator. User can write their own logic here.
-    :param model_version_info: Information about the model version which is in prediction. Its type can be
-                               ModelVersionMeta of py:class:`ai_flow.meta.model_meta.ModelVersionMeta`
-                               or Text. The model_version_info means version in the metadata service
-                               when its type is Text. The ai flow will get the model meta from metadata
-                               service by version.
-    :param output_num: The output number of the operator. The default value is 1.
-    :param name: Name of the predict operator.
-    :return: Channel or Tuple[Channel].
+    Predicts through the user-defined processor with the information of model and model version based on the input
+    :class:`~ai_flow.graph.channel.Channel`. The processor generates the model prediction
+    :class:`~ai_flow.ai_graph.ai_node.AINode`.
+
+    :param input: List of the input :class:`~ai_flow.graph.channel.Channel` that contains the multiple channels from the
+                           operators which generate the data used to predict model.
+    :param model_info: Information about the model predicted by the processor. The type of model could be
+                                     :class:`~ai_flow.meta.model_meta.ModelMeta` or Text(i.e. the name of model) or int(
+                                     i.e. the id of model). If the type is Text or int, this method will get the
+                                     metadata of model by the name or id of model by default.
+    :param prediction_processor: The user-defined processor composed of the logic that predicts model.
+    :param model_version_info: Information about the model version predicted by the processor. The type of model version
+                                                  could be :class:`~ai_flow.meta.model_meta.ModelVersionMeta` or Text(
+                                                  i.e. the version of model version). If the type is Text, this method
```

Comment:
`i.e. the version of model version ` -> `i.e. the version field of ModelVersionMeta`